tailwind.config.ts

```diff
@@ -6,36 +8,36 @@ const config: Config = {
     './src/components/**/*.{js,ts,jsx,tsx,mdx}',
     './src/app/**/*.{js,ts,jsx,tsx,mdx}',
     './src/containers/**/*.{js,ts,jsx,tsx,mdx}',
+    './src/styles/**/*.{js,ts,jsx,tsx,mdx}',
```

Comment:
Remove these 4 lines and add 1 line instead:

`'./src/**/*.{js,ts,jsx,tsx,mdx}'`